src/architecture/service-lifecycle.md

```diff
@@ -64,20 +64,38 @@ may diverge on different nodes of the network.
   with the external users in other ways. Different services instantiated
   from the same artifact are independent and have separate blockchain storages.
   Users can distinguish services by their IDs; both numeric and string IDs
-  are unique within a blockchain. Note that the transition to the "active" state
+  are unique within a blockchain. Note that the transition to the “active” state
   is not immediate; see [*Service State Transitions*](#service-state-transitions)
   section below.
 
-4. Active service instances can be stopped by a corresponding request to the core.
-  A stopped service no longer participates in business logic, i.e.,
-  it does not process transactions or hooks, and does not interact with the users
-  in any way. Service data becomes unavailable for the other services,
-  but still exists. The service name and identifier remain reserved
-  for the stopped service and can't be used again for adding new services.
+4. Active service instances can be *stopped* or *frozen* by a corresponding request
+  to the core.
 
 The core logic is responsible for persisting artifacts and services
 across node restarts.
 
+A **stopped** service no longer participates in business logic, i.e.,
+it does not process transactions or hooks, and does not interact with the users
+in any way. Service data becomes unavailable for the other services,
+but still exists. The service name and identifier remain reserved
+for the stopped service and can't be used again for adding new services.
```

Comment:
I am not familiar with the docs on resuming, but would it make sense saying that a stopped service may be resumed (with the same name)?

Reply:
Yeah, good idea 👍

Reply:
Done in 3b27163.